### ISSUE TITLE
[Auditbeat] [6.8] Linux build flaky integration test

### ISSUE THREAD

**Issue by ycombinator** [elastic/beats#14014]
Looking at the build history in https://beats-ci.elastic.co/job/elastic+beats+6.8+multijob-linux/beat=auditbeat,label=linux-immutable/, some builds have failed with errors like this:

```
13:37:19 Cannot create container for service beat: invalid CapAdd: unknown capability: "CAP_AUDIT_READ"
13:37:19 make[1]: *** [integration-tests-environment] Error 1
13:37:19 make[1]: Leaving directory `/var/lib/jenkins/workspace/elastic+beats+6.8+multijob-linux/beat/auditbeat/label/linux-immutable/src/github.com/elastic/beats/auditbeat'
13:37:19 make: *** [testsuite] Error 2
```

However, other 6.8 Linux Auditbeat builds have succeeded. So there appears to be some flakiness here.

Also note that this is only happening with the 6.8 build. It is not observed on `master`.

**Comment by andrewkroh** [comment]
Whether it fails probably varies based on worker node OS. I recall fixing it on master by removing CAP_AUDIT_READ from Dockerfile. It's not really necessary since there's another privilege that we request that covers it.

**Comment by ycombinator** [comment]
Thanks @andrewkroh. I put up a backport PR (#14015) for your PR making the change in `master` (#13077).

**Comment by ycombinator** [comment]
Closed via #14015.
